features/20-exchange-access-token.feature

```diff
@@ -5,13 +5,20 @@ Feature: App exchanges authorization code for access token
         Given OAuth is enabled
         And I am not logged in
         When I ask for authorization
-        And I exchange my authorization code
+        And I exchange my authorization code via a POST request
         Then the response code should be 200
         And the JSON response will contain access_token
         And the JSON response will contain token_type
         And the JSON response will contain scope
         And the JSON response will contain patient
 
+    Scenario: We cannot use a "GET" request for retrieving an access token
+        Given OAuth is enabled
+        And I am not logged in
+        When I ask for authorization
+        And I exchange my authorization code via a GET request
+        Then the response code should be 405
```

Comment:
I might be less prescriptive on this even the 405 is technically correct, the main invariant that we want to enforce is that the result does succeed -- 4xx or 5xx